### ISSUE TITLE
Energy converters disable

### ISSUE THREAD

**Issue by Beltrano42** [mrxd399/Sky-of-grind-modpack#4]
Hy, energy converters are all disable in the last version.

**Comment by RianGomita** [comment]
Yup, known issue, GTm last update changed the name of the file. Feel free to enable them in config.

(Fixed soon for the UEV Tier Update).
